community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -204,10 +204,10 @@
             setting( "unsupported.dbms.transaction_start_timeout", DURATION, "1s" );
 
     @Description("The maximum time interval of a transaction within which it should be completed.")
-    public static final Setting<Long> transaction_timeout = setting( "dbms.transaction.timeout", DURATION, "0" );
+    public static final Setting<Long> transaction_timeout = setting( "dbms.transaction.timeout", DURATION, "60s" );
```

Comment:
My idea is for now that in 3.0 we continue to use 'enabled' property to have the same way how we enable guard. But in 3.1 we should be able to remove that.